### ISSUE TITLE
Modern Fortran in Science and Technology by Lindemann and Dahlblom

### ISSUE THREAD

**Issue by Beliavsky** [fortran-lang/webpage#102]
[Modern Fortran in Science and Technology](https://modern-fortran-in-science-and-technology.readthedocs.io/en/latest/) could be added to the Online Courses section. I skimmed it, and it seems pretty thorough.

**Comment by Naveen2752** [comment]
I agree with @Beliavsky and @KHUSHIJAIN910, and I would also like to propose adding an “Online Courses” section to the Fortran website to help learners access curated tutorials and resources. This could include [Lindemann's YouTube series] (https://www.youtube.com/channel/UCEw5fnPEmyjMfayaU9ZVmqQ) on Scientific Programming in Python and Fortran, which offers valuable educational content, and we can also add relevant learning materials to it.

Additionally, I suggest implementing a responsive design with clean, organized sections and a consistent color scheme to enhance the website’s usability and professionalism.

I would be happy to work on a pull request to address these ideas if approved. Please let me know your thoughts!
